test/chainrules.jl

```diff
@@ -160,8 +160,15 @@ using Zygote, Test, ChainRules
 
         @test (1,) == h(1)
 
-        a3, pb3 = Zygote.pullback(h, 1)
-        @test ((1,),) == pb3(1)
+        if VERSION >= v"1.6-"
+            @test_broken begin
+                a3, pb3 = Zygote.pullback(h, 1)
+                ((1,),) == pb3(1)
+            end
```

Comment:
what is broken here?

Reply:
Really not sure. That's the one I mentioned in the description, which doesn't occur when run with JuliaInterpreter. I didn't have time yet to really look into it, but this might be a Julia bug.

Reply:
Nested AD is hard

Reply:
Yeah. Diffractor.jl to the rescue! :)

Reply:
Not entirely.
But a but.
The code will be *less* gross and a bit faster.
But the problem is just legit hard.